src/bid/bid.rs

```diff
@@ -60,6 +59,81 @@ impl Borrow<u64> for Bid {
     }
 }
 
+impl PartialEq for Bid {
+    fn eq(&self, other: &Self) -> bool {
+        self.hash().eq(&other.hash())
+    }
+}
+
+// This needs to be between braces since const fn calls passed as const_generics
+// params aren't perfectly supported yet.
+impl Serializable<{ Bid::serialized_size() }> for Bid {
```

Comment:
Like @ZER0 mentioned on a few other PRs - would it not make sense to directly use the number here? It prevents the reader from having to look this up if they want to know what the size is by just looking at the trait. Could be wrong about this of course, but I remember this being brought up a few times :)

Reply:
If I leave the other function `Bid::serialized_size()` i allow people that doesn't need `dusk-bytes` as dep to get access to the serialized size. 

I don't think it's that bad. But open to suggestions @ZER0 @julesdesmit @vlopes11

Reply:
I believe we should centralize everything in `dusk-bytes` and reduce the code complexity. This way, I'd stick with @julesdesmit suggestion to just put the number there and remove any additional/unnecessary methods.

Some libraries such as plonk will have a broad set of users. That, likely, won't be the case of the blindbid - only Dusk should be a user. This way, its safe to assume the only possible usage is with `dusk-bytes`

Reply:
@CPerezz, I'm not saying you should remove that associated method - just to not use it for this trait impl, for the reasons mentioned. Does it make sense?

Reply:
Can I at least put the Constant? I mean, you don't always have your values as `32` but as: BlsScalar::SIZE + JubJubScalar::SIZE`.

If you want a constant for that and you compute it's size based on it's components sizes, yo uneed a const_fn. I think putting a raw number in these cases is error prone TBH.

But it's 2v1 so if you don't think that what I did makes sense, I'll simply apply your solution

Reply:
I'll let @ZER0 be the arbiter here. I'm simply going off what I saw in the past - maybe the opinion on this changed while I've been working on `dusk-blockchain`.